src/item_list.js

```diff
@@ -18,4 +18,4 @@ export default function getHtmlListFromArray(itemArray) {
   return htmlOutput;
 }
 
-export const htmlListItems = getHtmlListFromArray(items);
+export const htmlListItems = getHtmlListFromArray(updateQuality(items));
```

Comment:
Having multiple functions in a single line like this can be potentially confusing...I might suggest doing something like this:

```
const updatedItems = updateQuality(items);
export const htmlListItems = getHtmlListFromArray(updatedItems);
```
This function is interesting because you're taking items, putting them into a function (`updateQuality`) and _then_ passing the return value from _that_ into `getHtmlListFromArray` and returning the output of _that_. 

![image](https://user-images.githubusercontent.com/7015466/109662920-8f274580-7b39-11eb-8dcc-34967b747dc6.png)

This is piping (i.e., taking a value and putting it through an assembly line). One of the popular functional programming libraries is called Ramda, and there's a function (`pipe`) [for something like this](https://ramdajs.com/docs/#pipe). You don't need to use it here, but that's something you might find interesting. It means your function _could_ look something like this:
```
export const htmlListItems = pipe(
  updateQuality,
  getHtmlListFromArray,
  )(items);
 ```
Just something to be aware of!